conf/janus.plugin.audiobridge.cfg.sample

```diff
@@ -19,5 +22,8 @@
 description = Demo Room
 secret = adminpwd
 sampling_rate = 16000
+audiolevel_event = no
+audio_active_packets = 100
+audio_level_average = 25
```

Comment:
If the default for `audiolevel_event` is "no", I'd remove these three lines from the sample room configuration. They're already documented above so no need for a placeholder there.